test/rendition-mixin.test.js

```diff
@@ -135,35 +149,44 @@ QUnit.test('returns representations with width and height if present', function(
   assert.equal(renditions[2].height, undefined, 'rendition has a height of undefined');
 });
 
-QUnit.test('representations are disabled if their excludeUntil is after Date.now', function(assert) {
+QUnit.test('blacklisted playlists are not included in the representations list', function(assert) {
   let hlsHandler = makeMockHlsHandler([
     {
       bandwidth: 0,
-      excludeUntil: Infinity
+      excludeUntil: Infinity,
+      uri: 'media0.m3u8'
+    },
+    {
+      bandwidth: 0,
+      excludeUntil: 0,
+      uri: 'media1.m3u8'
     },
     {
       bandwidth: 0,
-      excludeUntil: 0
+      excludeUntil: Date.now() + 999999,
```

Comment:
That is true, and maybe in this case it would be overdoing it to add another one, but usually it is good to have a non 0 test, as 0 is falsy and often is an edge case.